blocks/library/latest-posts/index.js

```diff
@@ -88,9 +89,10 @@ registerBlockType( 'core/latest-posts', {
 			if ( ! latestPosts.length ) {
 				return (
 					<Placeholder
-						icon="update"
-						label={ __( 'Loading latest posts, please wait' ) }
+						icon="admin-post"
+						label={ __( 'Latest Posts' ) }
 					>
+					<Spinner />
```

Comment:
Weird that my linter didn't catch this. Thank you again. Also, can you tell I'm in need of a vacation? ;)